include/OnnxMlirRuntime.h

```diff
@@ -22,6 +22,7 @@
 #include <stdint.h>
 #endif
 
+#include <onnx-mlir/Runtime/OMAccelerator.h>
```

Comment:
Sorry I'm not understanding why we are exposing these calls to the user?

Reply:
They are part of the runtime as they are called by our code. Why exposed? If some system has time to initialized everything ahead of time and then want a very fast response time, then init is good. If some accel reserve a lot of resources once it is initialized, a user might want to free resources by uninitializing the device.

Reply:
Being part of runtime doesn't mean they have to be exposed to the users. We have lots of functions in the runtime that are not exposed to the users.

The time you save on the init call is all speculation. A typical inference call may take on the order of milliseconds. Do we expect the init call to take milliseconds? I doubt it. And the saving is only for the first call. As for freeing the resources, that should also be managed internally when the last thread exits the entry point like `run_main_graph`, etc.

The danger of exposing calls to users, particularly ones that are potentially unstable (due to the possibility of adding new accelerators), is that it will make changing the calls very difficult, e.g., if we find out later that the init/fini calls need to take a parameter for some accelerators. We don't add user visible API calls unless we have a high confidence that users must be involved. In this case, I'm not convinced that the users must be involved in the init/fini calls.

Reply:
Ok, will remove then the public interface for now; we can always change our minds later if we have a concrete need for it.